core/tests/puppeteer-acceptance-tests/spec/blog-admin-tests/assign-roles-to-users-and-change-tag-properties.spec.ts

```diff
@@ -40,29 +40,29 @@ describe('Blog Admin', function() {
   */
   it('should assign roles to users and change tag properties',
     async function() {
-      const guestUsr1 = await userFactory.createNewGuestUser(
+      const guestUsr1 = await userFactory.createNewUser(
         'guestUsr1', 'guest_user1@example.com');
-      const guestUsr2 = await userFactory.createNewGuestUser(
+      const guestUsr2 = await userFactory.createNewUser(
         'guestUsr2', 'guest_user2@example.com');
 
       await superAdmin.expectUserNotToHaveRole('guestUsr1', ROLE_BLOG_ADMIN);
-      await blogAdmin.assignUserToRoleFromBlogAdminPage(
+      await superAdmin.assignUserToRoleFromBlogAdminPage(
```

Comment:
Yes, I was sort of confused about the TODO since don't we need a super admin to expect the user to have a certain role say blog admin or blog post editor. In this case should we create a blog admin and super admin to run this test or is the blog admin supposed to view the roles the user has from the Blog Admin Page?

Reply:
Oh I missed the todo. TBH I don't understand it either. I think what is supposed to happen in this test is:

- Super admin grants blog admin role to blog admin user
- Blog admin can do all the things (give blog editor role to users, remove that role, etc.)

I don't think the blog admin page has a way to view the roles of users, so you might need to switch back to super admin for that. Probably also worth filing a TODO for the future to add that functionality to the blog admin page (or at least have some way to view a list of all blog editors) and fix the test accordingly.

Reply:
Done